src/main/java/com/example/bookapp/dto/request/CreateBookRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bookapp.dto.request;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    @NotBlank
+    private String title;
```

Comment:
two dtos are the same and have no plans of chaning later, i suggest following DRY here and creating one general Book RequestDto